scripts/ccg/align_tags_to_bert.py

```diff
@@ -0,0 +1,102 @@
+import sys
+import argparse
+import pandas as pd
+import os
+import json
+from ..src import utils
+from src.utils import retokenize
```

Comment:
This is a little confusing - either use relative imports or only rely on the symlink (probably safer)?
```
from src import utils
from src.utils import retokenize
```